run-unit-test.sh

```diff
@@ -15,5 +15,6 @@ virtualenv env
 . env/bin/activate
 pip install -e .
 pip install nose mock coverage
+rm .coverage
```

Comment:
This should be unnecessary, I've never seen coverage.py object to having an old coverage directory present.